authors/settings.py

```diff
@@ -15,21 +15,20 @@
 # Build paths inside the project like this: os.path.join(BASE_DIR, ...)
 BASE_DIR = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
 
-
-
 # Quick-start development settings - unsuitable for production
 # See https://docs.djangoproject.com/en/1.11/howto/deployment/checklist/
 
 # SECURITY WARNING: keep the secret key used in production secret!
 SECRET_KEY = os.getenv('SECRET_KEY')
```

Comment:
@denomash The SECRET_KEY is set from the environment. Please add an export for it under the test instructions on this PR. Also I did not need to add the other DB exports to view the documentation. I only had to do a migrate.

Reply:
Nice observation @jillo-abdullahi. Let me add that in the PR

Reply:
Awesome!